.github/workflows/retest_periodic.yml

```
@@ -23,4 +23,4 @@ jobs:
     - name: Retest
       env:
         GITHUB_TOKEN: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
-      run: cd tools/retest && go run main.go
+      run: cd tools/retest && go run ./...
```

Nit: I think it might be sufficient to:
```suggestion
      run: cd tools/retest && go run .
```

---

Correct but `./...` is future proof is somebody create a sub package